contracts/RewardEscrowV2.sol

```diff
@@ -355,41 +396,67 @@ contract RewardEscrowV2 is
             totalEscrowedAccountBalance[msg.sender] -= totalWithFee;
             totalVestedAccountBalance[msg.sender] += total;
 
-            // trigger event
-            emit Vested(msg.sender, total);
-
-            // Send any fee to Treasury
+            // Send 50% any fee to Treasury and
+            // 50% to EarlyVestFeeDistributor
+            // UNLESS Distributor isn't set
+            // then send all funds to Treasury
             if (totalFee != 0) {
-                /// @dev this will revert if the kwenta token transfer fails
-                kwenta.transfer(treasuryDAO, totalFee);
+                if (earlyVestFeeDistributor == address(0)) {
+                    kwenta.transfer(treasuryDAO, totalFee);
+                    emit EarlyVestFeeSentToDAO(totalFee);
+                } else {
+                    /// @dev this will revert if the kwenta token transfer fails
+                    uint256 proportionalFee = totalFee / 2;
+                    kwenta.transfer(treasuryDAO, proportionalFee);
+                    kwenta.transfer(earlyVestFeeDistributor, proportionalFee);
```

Comment:
As far as I can tell it won't cause any issues (other than tiny amounts of dust being left in the contract). However if we wanted to, we could add the logic to add 1 extra wei to the `treasuryDAO` or `earlyVestFeeDistributor` for odd amounts, but it would be a bit more gas - so not sure it's worth it.